kubernetes/structures.go

```diff
@@ -192,6 +218,18 @@ func isInternalKey(annotationKey string) bool {
 	return false
 }
 
+// This function verifies if a given Kubernetes metadata(annotations and labels) key
+// is among the keys that need to be ignored
+func isIgnoteKey(kubernetesMetadataKey string, ignoreKubernetesMetadata []string) bool {
```

Comment:
Looks like a typo – should be `ignoreKey`? I might even drop the `is` here and just call this `ignoreKey()`

Reply:
I also wouldn't be so explicit with the naming of the parameters, in principle this function is quite generic. It takes a string, and a slice of strings to be parsed as regular expressions and returns true if any of them match. So you could call the parameters `key` and `expressions`. I would also mention in the comment above that the slice should contain regular expressions.

Reply:
I like this suggestion, done!